### ISSUE TITLE
SC2016 gives false positive on regex parameters

### ISSUE THREAD

**Issue by Dmole** [koalaman/shellcheck#1199]
#### For bugs
- Rule Id:
SC2016
- version:
0.3.7
- I tried on shellcheck.net and verified that this is still a problem

#### Here's a snippet that shows the problem:

```sh

#!/bin/bash

rename 's/(.)a/$1/g' ./*

```

#### Here's what shellcheck currently says:

> Line 3:
> rename 's/(.)a/$1/g' ./*
>        ^-- SC2016: Expressions don't expand in single quotes, use double quotes for that.

#### Notes

rename has a false positive but perl does not;

```sh

#!/bin/bash

perl -pi -e 's/(.)a/$1/g' ./*


```

**Comment by koalaman** [comment]
Fixed by @ngzhian by whitelisting `rename` as a command where single quoted $s are probably expected. Thanks!
